### PR TITLE
Update gapic yaml with streaming pull settings

### PR DESCRIPTION
**Pull request by michaelbausor**

### CODE REVIEW

File: google/pubsub/v1/pubsub_gapic.yaml [hunk 1]
```diff
@@ -186,6 +186,20 @@ interfaces:
...
       subscription: subscription
     timeout_millis: 60000
+  - name: StreamingPull
+    required_fields:
+    - subscription
+    - stream_ack_deadline_seconds
+    grpc_streaming:
+      response:
+        resources_field: received_messages
+    request_object_method: true
```
Comment (garrettjonesgoogle) [inline google/pubsub/v1/pubsub_gapic.yaml]: a request object method doesn't make sense for a streaming method. Maybe we should even make this an error.

Comment (michaelbausor) [inline google/pubsub/v1/pubsub_gapic.yaml]: Fixed, and created issue: https://github.com/googleapis/toolkit/issues/850

**Comment by michaelbausor** [comment]
PTAL

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**